recipe/meta.yaml

```
@@ -1,22 +1,25 @@
 {% set name = "accelerate" %}
-{% set version = "0.21.0" %}
+{% set version = "0.33.0" %}
 
 package:
   name: huggingface_{{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/accelerate-{{ version }}.tar.gz
-  sha256: e2959a0bf74d97c0b3c0e036ed96065142a060242281d27970d4c4e34f11ca59
+  sha256: 11ba481ed6ea09191775df55ce464aeeba67a024bd0261a44b77b30fb439e26a
 
 build:
   number: 0
   skip: true  # [py<38]
+  # s390x is missing safetensors for 0.4.4
+  skip: true  # [linux and s390x]
   # ppc64le is missing pytorch for 3.11
   skip: true  # [py>310 and (linux and ppc64le)]
   entry_points:
     - accelerate=accelerate.commands.accelerate_cli:main
     - accelerate-config=accelerate.commands.config:main
+    - accelerate-estimate-memory=accelerate.commands.estimate:main
     - accelerate-launch=accelerate.commands.launch:main
```

```suggestion
    - accelerate-launch=accelerate.commands.launch:main
    - accelerate-merge-weights=accelerate.commands.merge:main   
```
https://github.com/huggingface/accelerate/blob/v0.33.0/setup.py#L68